fusor-ember-cli/app/routes/review/progress/overview.js

```diff
@@ -3,12 +3,11 @@ import Ember from 'ember';
 export default Ember.Route.extend({
   model: function () {
       var deployment = this.modelFor('deployment');
-      var deployTaskPromise = this.store.find('foreman-task', deployment.get('foreman_task_uuid'));
+      var deployTaskPromise = this.store.find('foreman-task', {search: "id = " + deployment.get('foreman_task_uuid')});
```

Comment:
This is the main fix -- to use scoped search to get the task where rather than `find()` by id.
From what I think was happening, Ember Data said "hey, I already have this id in the `store`". I'm not going to ask the server for it.